src/gsy_e/models/area/coefficient_area.py

```diff
@@ -36,16 +36,18 @@ class CoefficientAreaException(Exception):
     """Exception that is raised when serializing an area."""
 
 
-def assert_coefficient_area_settings(
-        strategy: Optional[SCMStrategy], setting: Optional[float], setting_name: str) -> float:
+def validate_coefficient_area_setting(
+        children: Optional[List], setting: Optional[float], setting_name: str) -> float:
     """Check if coefficient area that is not an asset provided SCM setting."""
-    if strategy is None and setting is None:
-        raise CoefficientAreaException(f"In SCM simulations {setting_name} can not be None.")
+    if children:
+        if all(not child.children for child in children) and setting is None:
```

Comment:
As far as I see there is a method called `_is_home_area` in the CoefficientArea class, can it not be used here too ?

Reply:
You were mentioning this in the original PR already IIRC, but I must have ignored it. Sorry.
I adapted it here:[0bfff14](https://github.com/gridsingularity/gsy-e/pull/1551/commits/0bfff1456706bc89ea43f2019b71e597d5457d43)
I realised, that I can also move the check for the grid_fees before the init of the super class. Thanks a lot!